crates/bindings-csharp/Codegen.Tests/fixtures/server/Lib.cs

```diff
@@ -143,3 +143,25 @@ public static void Init(ReducerContext ctx)
         );
     }
 }
+
+[Table(Name = "MultiTable1", Public = true)]
+[Table(Name = "MultiTable2")]
+public partial struct MultiTable
```

Comment:
Nitpick; could we name this one RowType or something?

ie making it clear the distinction between the struct Foo being the row type, and the [Table(Name = "Foo")] being the table defined from that row type.

Reply:
Done